### PR TITLE
Fixed 5 code lint errors

### PR DESCRIPTION
**Pull request by samyakbvs**
issue #6

### CODE REVIEW

**Comment by samyakbvs** [comment]
Sorry I can't squash the commits in this PR

**Comment by samyakbvs** [comment]
I hope you guys will understand 👍

File: js/levels.js [hunk 1]
```diff
@@ -7,6 +7,6 @@ function createTestLevel()
     [door.none, door.top, door.both, door.both, door.both, door.top],
     [door.none, door.top, door.both, door.both, door.both, door.top],
     [door.none, door.top, door.both, door.both, door.both, door.top],
-    [NullTile, door.none, door.none, door.none, door.none, door.none],
+    [door.none, door.top, door.both, door.both, door.both, door.top],
```
Comment (abishekvashok) [inline js/levels.js]: Don't change this.

**Comment by samyakbvs** [comment]
Pls merge

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**